geometry_msgs/Point position
string ip
uint32 x
uint32 y
nav_msgs/OccupancyGrid  map 